spec/cassia/access_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+
+RSpec.describe Cassia::AccessController do
+  include FaradayHelpers
+  
+  describe "#get_token" do
+    vcr_options = { cassette_name: 'access_controller/get_token/success', record: :new_episodes }
+      context "when successful", vcr: vcr_options do
+        it "sets the access_token" do
+          access_controller = described_class.new
```

Comment:
one minor note, try and split your testing code up a little instead of having them one line after another. Generally I prefer to do
```
setup

action

expectations
```

so

```
access_controller = described_class.new

access_controller.get_token

expect(access_controller.access_token).not_to be_nil
```

Can you try and do that for all of the tests in this file?